packages/compass-explain-plan/src/components/explain-plan-side-summary.tsx

```diff
@@ -0,0 +1,251 @@
+  useDarkMode,
+  cx,
+  IndexBadge,
+} from '@mongodb-js/compass-components';
+import type { SerializedExplainPlan } from '../stores/explain-plan-modal-store';
+
+const defaultFormatter = (_: unknown) => String(_);
+
+const statsStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[2],
+});
+
+const statsTextStyles = css({
+  lineHeight: '20px',
+});
+
+const ExplainPlanSummaryStat = <T extends string | boolean | number>({
+  as = 'li',
+  glyph,
+  value,
+  formatter = defaultFormatter,
+  label,
+  definition,
+}: {
+  as?: keyof ReactHTML;
+  glyph?: ReactElement;
+  value?: T;
+  formatter?: (val: T) => string;
+  label: ReactNode;
+  definition: string;
+}): ReactElement | null => {
+  return React.createElement(
+    as,
+    { className: statsStyles },
+    glyph
+      ? React.cloneElement(glyph, {
+          viewBox: `0 0 ${spacing[3]} ${spacing[3]}`,
+          width: spacing[3],
+          height: spacing[3],
+        })
+      : null,
+    <InlineDefinition
+      definition={definition}
+      className={statsTextStyles}
+      tooltipProps={{ align: 'left', spacing: spacing[3] }}
+    >
+      {typeof value === 'undefined' ? (
+        label
+      ) : (
+        <>
+          <strong>{formatter(value)}</strong>&nbsp;{label}
+        </>
+      )}
+    </InlineDefinition>
+  );
+};
+
+type ExplainPlanSummaryProps = {
+  docsReturned: number;
+  docsExamined: number;
+  executionTimeMs: number;
+  sortedInMemory: boolean;
+  indexKeysExamined: number;
+  indexType: SerializedExplainPlan['indexType'];
+  indexKeys: [string, unknown][];
+};
+
+const summaryCardStyles = css({
+  display: 'grid',
+  gridTemplateRows: 'auto 1fr',
+  maxHeight: '100%',
+  overflow: 'hidden',
+});
+
+const summaryHeadingStyles = css({
+  paddingTop: spacing[3],
+  paddingBottom: spacing[3],
+  paddingLeft: spacing[4],
+  paddingRight: spacing[4],
+  backgroundColor: palette.gray.light3,
+  boxShadow: `0 0 0 1px ${palette.gray.light2}`,
+});
+
+const summaryHeadingDarkModeStyles = css({
+  backgroundColor: palette.gray.dark3,
+  boxShadow: `0 0 0 1px ${palette.gray.dark2}`,
+});
+
+const statsListStyles = css({
+  display: 'grid',
+  gap: spacing[3],
+  paddingTop: spacing[3],
+  paddingBottom: spacing[4],
+  paddingLeft: spacing[4],
+  paddingRight: spacing[4],
+  overflow: 'auto',
+});
+
+const indexesSummaryStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  gap: spacing[2],
+});
+
+export const ExplainPlanSummary: React.FunctionComponent<
+  ExplainPlanSummaryProps
+> = ({
+  docsReturned,
+  docsExamined,
+  executionTimeMs,
+  sortedInMemory,
+  indexKeysExamined,
+  indexType,
+  indexKeys,
+}) => {
+  const darkMode = useDarkMode();
+
+  const indexMessageText = useMemo(() => {
+    const typeToMessage = {
+      COLLSCAN: 'No index available for this query.',
+      COVERED: 'Query covered by index:',
+      MULTIPLE: 'Query used the following index (shard results differ):',
```

Comment:
This is `Shard results differ (see below)` in the explain plan for find, but this label doesn't make sense with the new summary anymore. Open to other suggestions here

Reply:
Took me a while to understand this comment and yes `below` does not make sense in this new view. 
nit: should it `indexes` as its multiple?

Reply:
Yeah, sorry, it is indeed hard to understand, I'll rephrase for anyone else who might stumble on this

> nit: should it be indexes as its multiple?

Yep, totally, good catch